### ISSUE TITLE
 42 miou was supposed to come out, but 35 miou came out

### ISSUE THREAD

**Issue by KimtaeHo3948** [open-mmlab/mmsegmentation#1965]
i'm not using  mmsegmentation pipeline but i run the pretrained model which came from mmsegmentation.
 but the performance did not come out. 42 miou was supposed to come out, but 35 miou came out. 
How do you evaluate?

i download pretrained model from https://github.com/open-mmlab/mmsegmentation/tree/master/configs/upernet
(upernet_r101_512x512_80k_ade20k_20200614_185117-32e4db94.pth)

Please explain in detail.
in my case, 
, only normalization of the ade20k validation image is fired and put into the model. Resize seg_logit to the original image size and make predictions through argmax.

Can you tell me specifically how the image is processed during evaluation and how the seg_logit is processed?

![스크린샷 2022-08-24 오후 8 54 49](https://user-images.githubusercontent.com/93563329/186414670-39f8ac67-463d-48ae-8ee1-b15afe856ddb.png)
![스크린샷 2022-08-24 오후 9 03 34](https://user-images.githubusercontent.com/93563329/186414724-6de32494-210c-4524-9e23-b46faef507aa.png)
![스크린샷 2022-08-24 오후 9 10 38](https://user-images.githubusercontent.com/93563329/186414845-115a0674-3efd-4441-a7f3-92a365accd8f.png)

**Comment by MeowZheng** [comment]
I think the performance reduction is caused by removing resize, as pre-trained models in mmseg follow the data augmentation setting with resize and we test them with resize also

https://github.com/open-mmlab/mmsegmentation/blob/dd42fa8d0125632371a41a87c20485494c973535/configs/_base_/datasets/ade20k.py#L10

https://github.com/open-mmlab/mmsegmentation/blob/dd42fa8d0125632371a41a87c20485494c973535/configs/_base_/datasets/ade20k.py#L23

**Comment by KimtaeHo3948** [comment]
thanks @MeowZheng 
when validating
1.Resize(img_scale=(2048, 512), keep_ratio=true)
2.Normalize(mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375], to_rgb=True)

model output: (batch_size, num_classes, (Resize_shape)h, (Resize_shape )w)

and we resize model output(e.g., seg_logit)
to (batch_size, num_classes, (original_shape)H, (original_shape)W)

and evaluating? 

Is what I said right now correct?

**Comment by KimtaeHo3948** [comment]
Hi @MeowZheng
i 've done resize but still do not get performace.
Is it correct to get the categorical class iou list through the mean_iou code and then do nanmean to get the total miou?
Is it correct to then average over all vlidation images?
